src/reducers/components.js

```diff
@@ -21,6 +21,11 @@ const components = (state = {}, action) => {
       };
     }
 
+    case 'COMPONENT_REMOVE': {
+      const { [action.payload.id]: _, ...newState } = state;
```

Comment:
At least for this file, can we open up the curly braces instead of putting everything on the same line? Eventually as the project grows we can add prettier.

Reply:
I'm not too sure about opening up the braces on an object destructuring statement for a single-key removal, it places the assignment at the bottom. 

```js
const {
  [action.payload.id]: _,
  ...newState
} = state;
```

Maybe it's just that I'm not used to it. How strongly do you feel about this format change?

Reply:
I think the format you've provided looks okay. It's not really a normal assignment with the spread syntax anyways.

Reply:
I'll open up the braces for consistency, it's probably better in the long run, especially when enforcing it through prettier.